### ISSUE TITLE
Failure to launch NLP suite after installing "Stanza" package in Python

### ISSUE THREAD

**Issue by shawnchen03** [NLP-Suite/NLP-Suite#1368]
The NLP suit returns this when I try to open it. The NLP Suite wellcome console does not show up.
![123](https://github.com/NLP-Suite/NLP-Suite/assets/140224372/2a78751f-4979-4bd2-865f-8811eadeaf97)

Previously, I have done setting up all the external software and the I/O settings in the Suite. The only thing that I need to do is check this box that requires me to install Stanza. 
<img width="676" alt="图片_20240125145037" src="https://github.com/NLP-Suite/NLP-Suite/assets/140224372/772b0706-1a90-4d91-9770-2e8107b77e86">
<img width="362" alt="5ef1e950bec9ef1c45af0b4571c8f30" src="https://github.com/NLP-Suite/NLP-Suite/assets/140224372/b4bec6ae-f694-4a16-9d10-9e35624fe815">

So I did, following exactly how https://github.com/NLP-Suite/NLP-Suite/issues/1360 suggested. Then NLP suite ran into this problem.

**Comment by shawnchen03** [comment]
The problem fixed with help from Simon. Turns out I need to reinstall multiple packages.  After cd ed into the src directory, I used pip install -r requirements.txt. Then was able to open the NLP suite.

**Comment by sengaungsein** [comment]
<img width="995" alt="Screenshot 2024-01-28 at 8 07 25 PM" src="https://github.com/NLP-Suite/NLP-Suite/assets/145607641/273a88f4-91ca-4147-9a07-1cbc46cd4129">

I am unsure if this is the same issue, but there is still no download after several hours.

**Comment by sengaungsein** [comment]
> <img alt="Screenshot 2024-01-28 at 8 07 25 PM" width="995" src="https://private-user-images.githubusercontent.com/145607641/300308532-273a88f4-91ca-4147-9a07-1cbc46cd4129.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY1MDMyNzUsIm5iZiI6MTcwNjUwMjk3NSwicGF0aCI6Ii8xNDU2MDc2NDEvMzAwMzA4NTMyLTI3M2E4OGY0LTkxY2EtNDE0Ny05YTA3LTFjYmM0NmNkNDEyOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyOVQwNDM2MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05YzJlOTc3Y2Y1YjNjNWExZmE4MmU2ODBjNjg4MzAyNWM1ODNiN2E0NWZkZWY0NDRmYjE0NDJiYjBjOGMwZWRiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.zZoN0_BA2-Z2eOFW_FRU-nx6u5HLbuYG5QqTIwrTVXY">
> I am unsure if this is the same issue, but there is still no download after several hours.

I just found out that you just need to wait forever. And finally it works.

**Comment by shawnchen03** [comment]
> > <img alt="Screenshot 2024-01-28 at 8 07 25 PM" width="995" src="https://private-user-images.githubusercontent.com/145607641/300308532-273a88f4-91ca-4147-9a07-1cbc46cd4129.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY1MDMyNzUsIm5iZiI6MTcwNjUwMjk3NSwicGF0aCI6Ii8xNDU2MDc2NDEvMzAwMzA4NTMyLTI3M2E4OGY0LTkxY2EtNDE0Ny05YTA3LTFjYmM0NmNkNDEyOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyOVQwNDM2MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05YzJlOTc3Y2Y1YjNjNWExZmE4MmU2ODBjNjg4MzAyNWM1ODNiN2E0NWZkZWY0NDRmYjE0NDJiYjBjOGMwZWRiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.zZoN0_BA2-Z2eOFW_FRU-nx6u5HLbuYG5QqTIwrTVXY">
> > I am unsure if this is the same issue, but there is still no download after several hours.
> 
> I just found out that you just need to wait forever. And finally it works.

Yes, its just downloading a lot of stuff I guess. Simon was able to help us with this problem. He said this is because one of the packages is big.
To solve this you should go the the requirements.txt in your NLP's src folder. At line 98 you will find the word "Tensor". Remove it from the text and then run the pip install -r requirements.txt in the NLP environment.
I
